<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>pankki</class>
 <widget class="QDialog" name="pankki">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>396</width>
    <height>538</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>50</y>
     <width>341</width>
     <height>431</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="sizeConstraint">
     <enum>QLayout::SizeConstraint::SetDefaultConstraint</enum>
    </property>
    <item>
     <widget class="QLabel" name="label">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>500</width>
        <height>50</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Pankki</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButtonSaldo">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Fixed"/>
      </property>
      <property name="maximumSize">
       <size>
        <width>700</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LayoutDirection::LeftToRight</enum>
      </property>
      <property name="text">
       <string>Näytä saldo</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButtonTilitap">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Fixed"/>
      </property>
      <property name="text">
       <string>Selaa tilitapahtumia</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="talletaRahaaButton">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Fixed"/>
      </property>
      <property name="text">
       <string>Talleta rahaa</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="nostaRahaaButton">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Fixed"/>
      </property>
      <property name="text">
       <string>Nosta rahaa</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButtonTili">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Fixed"/>
      </property>
      <property name="text">
       <string>Näytä tilin tiedot</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButtonAsiakas">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Fixed"/>
      </property>
      <property name="text">
       <string>Näytä tiedot</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButtonKortti">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Fixed"/>
      </property>
      <property name="text">
       <string>Näytä kortin tiedot</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="kirjauduUlosButton">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Fixed"/>
      </property>
      <property name="text">
       <string>Kirjaudu ulos</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
